openedx/features/learner_profile/toggles.py

```diff
@@ -12,7 +12,7 @@
 
 # Waffle flag to redirect to another learner profile experience.
 # .. toggle_name: REDIRECT_TO_PROFILE_MICROFRONTEND
-# .. toggle_type: waffle_flag
+# .. toggle_implementation: WaffleFlag
```

Comment:
Same request for name. 
I wonder if toggle_key, or something like that, would make it less likely to be an issue? Or, maybe the other linter could be enhanced to help with this. Future idea.